### DISCUSSION TITLE
Need help with setting up Angular/Vuejs CI/CD Pipeline

### DISCUSSION THREAD

**Discussion by arunprasathv** [concourse/concourse#6178] [Help & Support]
Hello, @evanphx 

I need to setup vue.js concourse pipeline. Is there any sample to learn how to do it, its similar to angular. 
have to do npm install, npm build to create dist folder. 

Appreciate your help!.

**Comment by evanphx** [comment]
Please don’t tag random people, it’s very rude.

**Reply by arunprasathv** [reply]
Sorry..not sure whom to address.
